requirements-minimal.txt

```diff
@@ -6,7 +6,7 @@ django-bootstrap-form
 django-ipware
 django-mathfilters
 django-redis
-django>=1.10,<1.10.999
+django~=1.10
```

Comment:
> The problem is that 1.11.0a1 (an alpha release) satisfies that version spec :(

Wow, that just makes me cringe. I won't actually merge this particular change since it doesn't accomplish anything; basically I was curious.